openedx/core/djangoapps/discussions/discussions_apps.py

```diff
@@ -0,0 +1,79 @@
+from typing import List, Optional
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.http import HttpRequest
+from opaque_keys.edx.keys import LearningContextKey
+
+from openedx.core.lib.plugins import PluginManager
+
+DISCUSSION_APPS_NAMESPACE = 'openedx.discussion_apps'
+
+
+class DiscussionAppCapabilities:
+    """ Enum that lists capabilities supported by a discussion tool. """
+    LTI1p1 = u"lti1.1"
+    LTI1p3 = u"lti1.3"
+    IN_CONTEXT_DISCUSSIONS = u"in_context_discussions"
+    NOTIFICATIONS = u"notifications"
+    COHORTS = u"cohort_aware"
+
+
+class DiscussionApp:
```

Comment:
Should this be an abstract class?

Reply:
It won't help since unless we put this in an external repo, plugins won't import this, so it's more of a guide.